services/QuillLMS/spec/models/activity_spec.rb

```diff
@@ -877,4 +877,38 @@
       it { expect(subject).to eq(activity.data) }
     end
   end
+
+  describe 'diagnostic repeatable behavior' do
+    context 'when activity is diagnostic' do
+      let(:diagnostic_classification) { create(:diagnostic) }
+
+      it 'defaults repeatable to false on new record' do
+        diagnostic_activity = build(:activity, classification: diagnostic_classification)
+        expect(diagnostic_activity.repeatable).not_to be
```

Comment:
Since repeatable is a boolean attribute, you can use the helper:

`expect(diagnostic_activity).not_to be_repeatable` which I think reads more fluidly.